iothub/device/src/Transport/Amqp/AmqpAuthenticationRefresher.cs

```diff
@@ -12,13 +12,14 @@ namespace Microsoft.Azure.Devices.Client.Transport.Amqp
 {
     internal class AmqpAuthenticationRefresher : IAmqpAuthenticationRefresher
     {
-        private static readonly string[] s_accessRightsStringArray = AccessRightsHelper.AccessRightsToStringArray(AccessRights.DeviceConnect);
+        private static readonly string[] s_accessRightsStringArray = new[] { "DeviceConnect" };
```

Comment:
All the code in AccessRights.cs was doing was creating an array with one element, which we can hard-code here.